dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresServiceCollectionExtensions.cs

```diff
@@ -50,28 +52,16 @@ public static IServiceCollection AddPostgresVectorStore(this IServiceCollection
     /// <returns>The service collection.</returns>
     public static IServiceCollection AddPostgresVectorStore(this IServiceCollection services, string connectionString, PostgresVectorStoreOptions? options = default, string? serviceId = default)
     {
-        string? npgsqlServiceId = serviceId == null ? default : $"{serviceId}_NpgsqlDataSource";
-        // Register NpgsqlDataSource to ensure proper disposal.
```

Comment:
with a ctor that takes the connection string, there is no need to register data source anymore